drivers/ukconsole/ns16550/Config.uk

```diff
@@ -1,23 +1,28 @@
+# Port-IO mode for IBM PC/AT compatible machines
+config HAVE_IBMPC_NS16550
+	bool
+	depends on ARCH_X86_64
```

Comment:
I wonder if `depends` is necessary here, given that it's an invisible option, and given that in certain platforms (Firecracker) we select that option conditionally to the architecture anyway (i.e. `select HAVE_IBMPC_NS16550 if ARCH_X86_64`). Perhaps we can make all selections of `IBMPC` conditional to `ARCH_X86_64` at the platform configuration?

Reply:
Hum.. I think it is a question of how complete we are here. The thing what I wanted to express is that `HAVE_IBMPC_NS16550` is a pure x86 standard because it implies `portio` instructions at specific addresses.

Reply:
> The thing what I wanted to express is that HAVE_IBMPC_NS16550 is a pure x86 standard because it implies portio instructions at specific addresses

That also makes sense, as this is the authoritative source of information for this config option. My only concern is that it will look very strange on the platform's `Config.uk` without the condition of `ARCH_X86_64`. Saying that,  I think it won't hurt if  people define it conditional to `ARCH_X86_64` on the platform's `Config.uk` too. If you agree, let's leave it as is here, and add a condition at the platform's side too.

Reply:
Actually it looks like you're doing that already, so it's all good. Please feel free to resolve.
```
plat/kvm/Config.uk:	select HAVE_IBMPC_NS16550 if ARCH_X86_64
```